bool running
nav_msgs/Path path
float64[] gains

---
bool success
